soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -24,9 +27,21 @@ config LOG_BACKEND_SWO_REF_FREQ_HZ
 endif # LOG_BACKEND_SWO
 
 # set the tick per sec as a divider of the LPTIM clock source
+if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
```

Comment:
What about using `depends on STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE` instead of  `if` ?

Reply:
same result, changing to "depends on"